docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -337,31 +348,28 @@ This setup uses host-specific thresholds that are stored in PostgreSQL as a basi
           --kafka-topic cpu_load_stats_real_filter_pg                 \
           --schema-sql "TABLE_SQL"
 
-   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
-
-   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
-      :lines: 62-66
-      :language: sql
-
 #. Run the following command to list the tables for the ``demo-flink`` service and get the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables:
 
    ::
 
       avn service flink table list demo-flink
 
-#. Run the following command to create a data pipeline job named ``simple_filter_pg``:
+#. Run the following command to create a data pipeline job named ``simple_filter_pg``.
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/pgthresholds_job.md
+      :language: sql
 
    ::
 
       avn service flink job create demo-flink simple_filter_pg            \
           --table-ids CPU_IN_ID CPU_OUT_FILTER_PG_ID SOURCE_THRESHOLDS_ID \
           --statement "JOB_SQL"
 
-   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables, and ``JOB_SQL`` with the following:
+   The new job is added and starts automatically once a task slot is available.
 
-   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
-      :lines: 69-77
-      :language: sql
+   When the job is running, you should start to see messages indicating hosts with high CPU loads in the ``cpu_load_stats_real_filter_pg`` topic of your ``demo-kafka`` service.
```

Comment:
I would rephrase here like 
"indicating hosts with CPU loads overcoming the PostgreSQL defined thresholds in the ``cpu_load_stats_real_filter_pg`` topic of your ``demo-kafka`` service."